runtime-config-linux.md

```diff
@@ -146,7 +146,7 @@ For more information, see the [kernel cgroups documentation](https://www.kernel.
 
 The path to the cgroups can be specified in the Spec via `cgroupsPath`.
 `cgroupsPath` is expected to be relative to the cgroups mount point.
-If not specified, cgroups will be created under '/'.
+If not specified or specified as empty string, implementations can define the default cgroup path.
```

Comment:
On Wed, Nov 18, 2015 at 10:23:39AM -0800, Vish Kannan wrote:
> > -If not specified, cgroups will be created under '/'.
> > +If not specified or specified as empty string, implementations can define the default cgroup path.
> 
> This is specifically for developer workflow.

It doesn't read that way to me.  “implementations can define the
default cgroup path” sounds like “if you don't set cgroupsPath, we'll
pick one for you” (which I think makes sense).  However, “behavior is
undefined” sounds like “the runtime can do *anything* it wants,
including just exiting with an error and not launching a container at
all” (which I think makes very little sense).